.github/workflows/radar-pre-release-actions.yml

```diff
@@ -14,6 +14,8 @@ jobs:
           registry-url: 'https://registry.npmjs.org'
       - run: npm ci
       - run: npm run check-beta-tag
+      - run: npm run build
+      - run: npm run copy-assets
```

Comment:
we change the the folder that serves as the 'root' of the package from the `src` folder to the `dist` folder. We need to copy the assets over so that all the relative imports still work. The alternative is to use a package bundler and add another dev dependency.